### PR TITLE
t0220: add 'ipfs bitswap stat' tests

### PR DESCRIPTION
**Pull request by chriscool**
License: MIT
Signed-off-by: Christian Couder <chriscool@tuxfamily.org>

### CODE REVIEW

**Comment by chriscool** [comment]
It is strange because sometimes the result from `ipfs bitswap stat` contains `dup data received: 0B` and sometimes `dup data received: 0`.

**Comment by chriscool** [comment]
It looked like the Travis CI push had passed but no it failed. It was my version of ipfs that was not uptodate.

**Comment by chriscool** [comment]
One of the test failures is "could not resolve host github.com".

**Comment by cryptix** [comment]
> The test failure is "could not resolve host github.com".

Kicked it a few times, passed now :)

**Comment by chriscool** [comment]
Thanks @cryptix!

**Comment by cryptix** [comment]
> It is strange because sometimes the result from ipfs bitswap stat contains dup data received: 0B and sometimes dup data received: 0.

Maybe you had conflicting version? The byte count should be humanized since #1890.

File: test/sharness/t0220-bitswap.sh [hunk 1]
```diff
@@ -18,6 +18,37 @@ test_expect_success "'ipfs block get' adds hash to wantlist" '
...
+	dup blocks received: 0
+	dup data received: 0B
+	wantlist [1 keys]
+		$NONEXIST
+	partners [0]
+EOF
+	test_cmp expected stat_out
+'
+
+test_expect_success "ipfs peer id looks good" '
```
Comment (whyrusleeping) [inline test/sharness/t0220-bitswap.sh]: oh this is smart.

**Comment by jbenet** [comment]
LGTM thanks @chriscool

**Pull request merged**